### PR TITLE
Fix a few repo bugs

### PR DESCRIPTION
**Pull request by slifty**
This PR fixes a bug with our linting and a bug with our test coverage reporting.

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ module.exports = {
       tsconfig: 'tsconfig.dev.json',
     },
   },
+	collectCoverageFrom: ["src/**/*.ts"],
```
Comment (bickelj) [inline jest.config.js:7]: Evil tab! :grin:

Comment (slifty) [reply jest.config.js:7]: See this is why I need linting 😂

**Review by bickelj — APPROVED** [review]

**Comment by slifty** [comment]
(Just because I'm morbidly curious about how our test coverage was impacted I'm gonna wait until codecov provides the coverage report before merging... IF IT EVER DOES)

**Comment by slifty** [comment]
I'm losing faith in codecov; maybe they have rate limits or something.  I'll just merge :(

**Pull request merged**